packages/dashboard/src/customization/propertiesSections/settings/section.tsx

```diff
@@ -22,18 +22,34 @@ export const SettingsSection = ({
   significantDigits: number | undefined;
   updateSignificantDigits: (newValue: number | undefined) => void;
 }) => {
-  const onSignificantDigitsChange: NonCancelableEventHandler<
-    InputProps.ChangeDetail
-  > = (event) => {
-    updateSignificantDigits(
-      isNumeric(event.detail.value) ? numberFromDetail(event) : undefined
-    );
+  const { control, clearErrors } = useForm<{
+    decimalPlaces: string;
+  }>({ mode: 'onChange' });
+
+  const [decimalPlacesValue, setDecimalPlacesValue] = useState<
+    number | undefined
+  >(significantDigits);
+
```

Comment:
Do we really need 3 stateful instances of the value of decimal places? We have the one coming in as `signficantDigits`, the `decimalPlaces` in the form state, and `decimalPlacesValue`. I think `decimalPlacesValue` could be avoided by referencing `field.value` in the render.

Reply:
I have remvoed `decimalPlacesValue` state.